provisioning/transport/mqtt/src/ProvisioningTransportHandlerMqtt.cs

```diff
@@ -160,17 +164,17 @@ private DeviceRegistrationResult ConvertToProvisioningRegistrationResult(Models.
                 {
                     ch.Pipeline.AddLast(
                         new ReadTimeoutHandler(ReadTimeoutSeconds),
-                        new TlsHandler(tlsSettings), //TODO: Ensure SystemDefault is used.
+                        new TlsHandler(stream => new SslStream(stream, true, (sender, certificate, chain, errors) => true), tlsSettings), //TODO: Ensure SystemDefault is used.
```

Comment:
Must remove. Please _never_ add this code without a very clear TODO that would avoid it being checked in.

Reply:
@alextolp custom certificate validation can be added via a public API similar to [this](https://github.com/dotnet/corefx/blob/f0a032e4a1c8a848574a5db3e69e8a6fa0aae91e/src/System.Net.Http/ref/System.Net.Http.cs#L101)